packages/trader/package.json

```diff
@@ -79,7 +79,7 @@
         "webpack-node-externals": "^2.5.2"
     },
     "dependencies": {
-        "@deriv/api-types": "^1.0.54",
+        "@deriv/api-types": "^1.0.11",
```

Comment:
here

Reply:
Because it is failing some of the test cases in codecov and it doesn't have that specific call unless you update it.